src/styles/_catalog.scss

```
@@ -1,22 +1,28 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  grid-template-columns: repeat(1, $card-width);
```

```suggestion
  grid-template-columns: repeat($columns, $card-width);
```

you may use variable